src/app/zap-templates/zcl/data-model/chip/rvc-clean-mode-cluster.xml

```diff
@@ -61,10 +61,5 @@ limitations under the License.
     </command>
   </cluster>
 
-  <bitmap name="Feature" type="bitmap32">
-    <cluster code="0x0055"/>
-    <field name="NoFeatures" mask="0x0"/>
-    <!-- Feature OnOff with code 0x01 is deprecated -->
-  </bitmap>
```

Comment:
I see no features for this cluster in new xml

Reply:
Yes, this cluster has no features, but the bitmap can't be empty and the bitmap needs to exist.  Hence the NoFeatures thing.

We need a way to continue supporting that.

Reply:
Yes, we had a feature that disappeared, so at some point we had a bitmap. We could not completely remove the bitmap because backwards compatibility and an empty bitmap generates code that refuses to compile (python and I believe even obj-c compiler complains). Hence we created a fake `NoFeatures` value.

Reply:
Added it back